mws/utils.py

```diff
@@ -66,7 +66,7 @@ def _parse_node(self, node):
             key, val = self._namespace_split(key, ObjectDict({'value': val}))
             node_tree[key] = val
         # Save childrens
-        for child in node.getchildren():
+        for child in node:
```

Comment:
This is a fairly basic upstream process that seems to have been working so far. Changing this now could have some nasty aftereffects without extensive testing.

@cmchecha , can you provide some detail here, please?

Reply:
Output in my production:
DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  for child in node.getchildren():

But we can return to the previous version.

Reply:
Ah yes, I see that now. Did some digging: `getchildren` is deprecated as of 2.7 ([doc](https://docs.python.org/2.7/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.getchildren)) and 3.2 ([doc](https://docs.python.org/3.6/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.getchildren)).

Spot-checked the change locally, and I confirm it works that same as original. We can accept that change. Good catch, thank you. :)